recipes/panic/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "panic" %}
+{% set version = "9.4.4" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: panic
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/panic-{{ version }}.tar.gz
+  sha256: 70a035f7618680a6bbf64dcaa56c3d25de510137eccd9fc89b1763dc65148cad
+  
+build:
+  number: 0
+  noarch: python
+
+requirements:
```

Comment:
We are not building any top level package, so I'd remove those requirements, maybe except (according to matplotlib recipe):

```
requirements:
  host:
    - python
```

And let the linter complain.

Or switch to recipe.yaml, because the format is much more explicit for multiple outputs :-)